docs/qs_first_boot.md

```diff
@@ -0,0 +1,101 @@
+# OVOS Quick Start - First Boot
+
+You now have a flashed boot medium with OVOS installed.  Now what?
+
+Insert your boot medium, and power on your device.
+
+**NOTE** On a RPi4 with a USB device, it is recommended to use the top-right USB3 port.
+
+Each image has a different experence when you first boot your device.
+
+[Buildroot - GUI](#Buildroot-GUI)
+
+[Raspbian-ovos - Headless](#Raspbian-ovos-Headless)
+
+**Notes on internet** While OVOS can run fully offline, with on device STT and TTS, your device will be lacking many of the things you expect from a smart assistant.  On a RPi, this includes the date and time because the Pi lacks a Real Time Clock, and therefor needs to connect to a server to set those on your device.
+
+## Buildroot - GUI
+
+When you first boot the Buildroot image, you will be greated with an OVOS splash screen as shown below.
+
+![](https://raw.githubusercontent.com/OpenVoiceOS/ovos_assets/master/Images/Screenshot%20-%20Buildroot%20bootsplash.png)
```

Comment:
```suggestion
![The splash screen is visible here.](https://raw.githubusercontent.com/OpenVoiceOS/ovos_assets/master/Images/Screenshot%20-%20Buildroot%20bootsplash.png)
```

Slow internet connections exist, and does accessibility software. Always good to remember that.